Doc/c-api/dict.rst

```diff
@@ -81,14 +81,15 @@ Dictionary Objects
 .. c:function:: int PyDict_DelItem(PyObject *p, PyObject *key)
 
    Remove the entry in dictionary *p* with key *key*. *key* must be hashable;
-   if it isn't, :exc:`TypeError` is raised.  Return ``0`` on success or ``-1``
-   on failure.
+   if it isn't, :exc:`TypeError` is raised, if *key* is not in the dictionary,
```

Comment:
I think this sentence would be more readable with a full stop after 'raised'.

Reply:
Done.